xayn-ai-ffi-wasm/src/ai.rs

```diff
@@ -648,7 +751,7 @@ mod tests {
             QAMBERT_VOCAB,
             QAMBERT_MODEL,
             LTR_MODEL,
-            Some(Box::new([2, 3, 4])),
+            Some(Box::new([u8::MAX])),
```

Comment:
I have no complain against the change.

But out of interest why was this changed?

Reply:
it better mirrors how this test is implemented in other places, and it's the "minimal" example - we just need the first byte to be an invalid version, so `u8::MAX` makes it comfortably so. then `2,3` or anything else is unnecessary.